src/main/java/randoop/generation/WeightedComponentManager.java

```diff
@@ -0,0 +1,53 @@
+package randoop.generation;
+
+import java.util.Collection;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import randoop.main.GenInputsAbstract;
+import randoop.sequence.Sequence;
+
+/**
+ * WeightedComponentManager extends the functionality of ComponentManager by adding needed
+ * functionality for weighted sequences and constants, for the command-line options
+ * --weighted-sequences and --weighted-constants.
+ */
+public class WeightedComponentManager extends ComponentManager {
+
+  /** Sequence frequency represents the number of times a sequence occurs in a set of classes */
```

Comment:
There seems to be a type error here.  A sequence cannot occur in a set of classes -- only a class can appear in a set of classes.  Can you clarify the comment?